sources/assets/exegol/load_supported_setups.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/zsh
 
+# Cannot source from a function
+source "$HOME/.zshrc"
```

Comment:
Nice, can you remove the other source then? I think it's a duplicate now isn't it?

Reply:
I'm not sure to understand. There is no duplicate, because I already removed the source from init().

That said, I looked again at the function init() and found I had left the corresponding colorecho. I removed as well now.

Reply:
My initial review was on the GitHub mobile app and it didn't show the removal of the initial source 🤷 thank you